src/test/java/com/google/gcp_variant_transforms/library/HeaderIteratorTest.java

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import com.google.gcp_variant_transforms.library.HeaderIterator;
```

Comment:
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

static imports should come first, then new line, then non-static imports. All sorted alphabetically.

Reply:
resolved